src/gbd_mapping_generator/data.py

```diff
@@ -324,10 +324,16 @@ def get_coverage_gap_data():
     out = _SPECIAL_COVERAGE_GAPS
 
     for c in gbd.get_coverage_gap_list():
-        metadata = get_coverage_gap_metadata(c)['exposure']
+        metadata = get_coverage_gap_metadata(c)
+
+        gbd_id = metadata['gbd_id'] if 'gbd_id' in metadata else None
```

Comment:
We should be able to assume that all the keys are present at this point (which is to say, push more stuff down into `gbd_access`.  This is fine for now though.

Reply:
yeah, will fix it a little later, for now I have too many branches on different repos.